awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupList.jsx

```diff
@@ -183,16 +198,12 @@ function InstanceGroupList({ i18n }) {
                 key={instanceGroup.id}
                 value={instanceGroup.name}
                 instanceGroup={instanceGroup}
-                detailUrl={`${match.url}/${instanceGroup.id}/details`}
+                detailUrl={verifyDetailUrl(instanceGroup)}
```

Comment:
I would change this to getDetailUrl, you're not really verifying the url you're returning it.

Reply:
Updated @jlmitch5. Thanks for your review.